src-api/source/includes/_projects.md

```diff
@@ -223,6 +230,72 @@ curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/tree/:br
 limit | The number of builds to return. Maximum 100, defaults to 30.
 offset | The API returns builds starting from this offset, defaults to 0.
 filter | Restricts which builds are returned. Set to "completed", "successful", "failed", "running", or defaults to no filter.
+shallow | An optional boolean value that may be sent to improve overall performance if set to 'true.'
+
+### Improving Performance In Recent Build Requests Using the `Shallow` Parameter
+
+When making API requests for information about recent builds, you may experience performance lag and a decrease in overall performance while the request is being processed by the server. To improve performance, CircleCI recommends you pass the `shallow` parameter in your request.
+
+The example below shows a user request for recent build information. Notice that when the user passes the `shallow` parameter, a limited set of information is returned, thereby improving response time and minimizing performance lag. 
+
+
+#### Sample Request Using the `Shallow` Parameter
```

Comment:
So, we actually want to move this in-line with the code examples by using the markdown quote block (`>`). You can see us doing it on line [188](https://github.com/circleci/circleci-docs/pull/3292/files#diff-960fbf6be43efb049b04a2a2ea6c46e6R188)

This ends up interspersing raw text in-between code examples in the right pane, as seen below.

![image](https://user-images.githubusercontent.com/12987958/56210230-5e13f980-6023-11e9-82fb-73eee12e395b.png)

The markdown source code to create the above formatting looks like:

![image](https://user-images.githubusercontent.com/12987958/56210497-f7dba680-6023-11e9-9ebc-4a5cc04ab1c6.png)

---


So we still need to add a new curl example that uses `shallow=true`. Adding something like the below should work:


```
curl https://circleci.com/api/v1.1/recent-builds?limit=1&shallow=true

> Sample Request Using the `Shallow` Parameter
```

Reply:
Fixed